src/calendar.ts

```diff
@@ -25,17 +25,87 @@ export const getMonth = (element: HTMLElement) => getTitleElement(element)?.inne
 export const getYear = (element: HTMLElement) => getTitleElement(element)?.innerText.replace(/.*\s([0-9]{4})$/, "$1");
 
 function getCurrentMonth(interactor: Interactor<HTMLElement, any>) {
-  return new Promise((resolve) => interactor.perform((element) => resolve(getMonth(element))));
+  return new Promise<string | undefined>((resolve) => interactor.perform((element) => resolve(getMonth(element))));
 }
 function getCurrentYear(interactor: Interactor<HTMLElement, any>) {
-  return new Promise((resolve) =>
+  return new Promise<number | undefined>((resolve) =>
     interactor.perform((element) => {
       const yearString = getYear(element);
       const year = yearString ? parseInt(yearString) : NaN;
       resolve(Number.isNaN(year) ? undefined : year);
     })
   );
```

Comment:
`perform` should really allow you to return a value out of it, I added an issue for this on bigtest: https://github.com/thefrontside/bigtest/issues/914